ogr/services/mock/mock.yaml

```diff
@@ -0,0 +1,4 @@
+generic:
```

Comment:
Why not have this in the code directly?

Reply:
My imagination is, to have there some templates, eg. as now there is integration testsuite stored `conu` and `colin` project, so that have there return for them.

And also I can imagine to use it also for storing values, in sense of some `--dryrun` with storing values on some internal place.

Reply:
> My imagination is, to have there some templates, eg. as now there is integration testsuite stored `conu` and `colin` project, so that have there return for them.

I'm sorry but I don't understand.

> And also I can imagine to use it also for storing values, in sense of some `--dryrun` with storing values on some internal place.

You can't be serious: changing files in sitelib? `rpm -V` would not be happy and this could result into some pretty bad errors or race conditions. Runtime data are meant to be stored elsewhere - also I don't understand why should a library store data.

Reply:
Ah sorry, this location is just for my purposes. It will not be here, I have it there just for debugging, then it could be stored beside tests, or whenever.